.github/workflows/ci.yml

```
@@ -57,6 +18,6 @@ jobs:
 
   release:
     name: Release on GitHub
-    needs: [ assert-branch, images ]
+    needs: [ images ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)